packages/table/src/table.tsx

```diff
@@ -357,10 +350,12 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         return columnIdToIndex;
     }
 
+    public grid: Grid;
+    public locator: Locator;
```

Comment:
`public` so they can be accessed in tests, although this is only marginally useful. can't make them `public readonly` because they're not assigned at initialization.